tests/draft-future/anchor.json

```diff
@@ -134,5 +134,40 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "same $anchor with different base uri",
+        "schema": {
+            "$id": "http://localhost:1234/root",
```

Comment:
This identifier is already used by another test; can you pick another?

(Side note to @Julian -- would it be possible to check for this in CI?)

Reply:
change `$id` to `http://localhost:1234/foobar`

Reply:
> (Side note to @Julian -- would it be possible to check for this in CI?)

I'm happy to implement that it definitely sounds possible, will open a ticket to track and stick it on my stack.